src/java.desktop/macosx/native/libawt_lwawt/awt/common.h

```diff
@@ -0,0 +1,159 @@
+/*
+ * Copyright (c) 2019, Oracle and/or its affiliates. All rights reserved.
```

Comment:
probably should be moved to the libawt_lwawt/java2d/metal/?

Reply:
Looks like  so

Reply:
JDK-8260681